BugSplat.h

```diff
@@ -75,6 +75,9 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, copy, nullable) NSString *userID;
 
+/** Set the bugsplat database name, which will override the "BugSplatDatabase" Info.plist key value */
+@property (nonatomic, assign) NSString *bugSplatDatabase;
```

Comment:
Should be copy instead of assign. Assign is for non-reference types, dangerous!

Reply:
Give the use case (immediate, read-only use and discard) the difference is negligeable, although the remark is technically correct.

Reply:
Sorry these were supposed to be internal notes